AddressBookApp/AddressBookApp/Common Code/MGCParsingUtilities.swift

```diff
@@ -0,0 +1,118 @@
+/*
+    Copyright (C) 2017 Apple Inc. All Rights Reserved.
+    See LICENSE.txt for this sample’s licensing information
+    
+    Abstract:
+    A utility class that allows us to parse the resource file
+    			containing the app's navigation menu.
+*/
+
+import UIKit
+
+class MGCParsingUtilities {
+    // MARK: - Types
```

Comment:
`MGCParsingUtilities` 클래스 사용하는 곳이 있나요? 검색해보니 저는 없는 것 같은데요? 
```suggestion
    // MARK: - Types
```
```suggestion
    // MARK: - Types
```

Reply:
사용하는 곳이 없는 파일들은 확인 후 제거하였습니다.